lib/firmware/signing.c

```diff
@@ -953,15 +953,6 @@ static bool signing_check_output(TxOutputType *txoutput) {
     send_fsm_co_error_message(co);
     signing_abort();
     return false;
-  } else if (co < 0) {
```

Comment:
these are dead code; `TXOUT_COMPILE_ERROR` is `0`, so the `co <= TXOUT_COMPILE_ERROR` branch above will always supersede these